how-we-work/oncall-responsibilities.md

```diff
@@ -0,0 +1,54 @@
+
+All engineers, both frontend and backend, will be on the on-call rotation to reduce the frequency that any one person is on call and to build shared ownership over our services.
+
+## Practice and Process
+
+An on-call rotation begins on Monday at 9am PST. Each week, there is a primary and secondary on-call engineer. You will get an email and Slack notification from PagerDuty (PagerDuty manages our on-call schedule) designating you as the on-call engineer. The manager will always be on tertiary on-call duty.
+
+### Pre-Requisites
+
+- Please make sure you have access to Sentry so that you can check out our error reporting. You can check if you have access by logging into https://czi-duo.okta.com/. If you do not have access, ask in the [#help-infra](https://chanzuckerbergteam.slack.com/archives/C94RQ5SBV) Slack channel.
+
+
+### Primary Responsibilities
+
+#### Deployment
+
+The principal responsibility of the primary on-call is to coordinate deployments on Wednesday (from `main` to `staging`) and Thursday (from `staging` to `prod`). Both the cellxgene Data Portal and the cellxgene Explorer needs to be deployed and require two different processes. On both days, please try to promote by 1pm PST.
+
+The instructions to deploy `hosted-cellxgene` (a.k.a. cellxgene Explorer) can be found [here](https://github.com/chanzuckerberg/single-cell-infra/tree/main/terraform/modules/hosted-cellxgene#redeploying-the-application). The instructions to deploy `corpora-data-portal` (a.k.a. cellxgene Data Portal) can be found [here](https://github.com/chanzuckerberg/single-cell-infra/tree/main/terraform/modules/corpora#redeploying-the-application).
+
+**On Wednesday**:
+
+- Send a note to the [#single-cell-ops](https://chanzuckerbergteam.slack.com/archives/CV7PFVABC) Slack channel to see if there are any PRs that you should wait on before the `main` to `staging` push. 
+- Once the deployment is in progress, let the team know in the same channel and also notify when the deployment is complete. Please write a quick note on the key new features/bug fixes for the engineering team to help test on staging. If the deployment fails, coordinate the fix.
+- Please manually test the cellxgene Explorer deployment, walking through the test cases in [this doc](https://docs.google.com/document/d/1nHdd8cDlmauv27oEemlMy_mEa0Dw7UMCp-w50IhNuK0/edit).
+
+**On Thursday**: 
+
+- Send a note to the [#single-cell-ops](https://chanzuckerbergteam.slack.com/archives/CV7PFVABC) Slack channel checking to make sure that engineers have had a chance to test the staging deployment. If you get a quorum of LGTMs, promote staging to prod. 
+- Let the channel know when the deployment is in progress and when it is complete. You are also responsible for coordinating any fixes if the deployment fails.
```

Comment:
```suggestion
- Let the channel know when the deployment is in progress and when it is complete. You are also responsible for rolling back to a working version and coordinating any fixes if the deployment fails.
```

Reply:
Do we have a written procedure on how to roll back?

EDIT: Looks like that will be written out in the incident playbook